spicy/toolchain/bin/spicy-dump/printer-text.cc

```diff
@@ -116,7 +136,11 @@ void TextPrinter::print(const type_info::Value& v) {
                     if ( y ) {
                         out() << '\n';
                         outputIndent();
-                        out() << f.name << ": ";
+
+                        if ( ! f.isAnonymous() )
+                            out() << f.name;
```

Comment:
I believe this is still inconsistent and not what we want. It seems that e.g., `spicy-dump` omits any anonymous field unless it is a bitfield, e.g., given the following unit
```ruby
public type X = unit {
    : uint8;
    : bitfield(8) { a: 0..7; };
};
```
I see the following output
```console
$ printf '\x00\x00' | spicy-dump bla.spicy
bla::X {
  : (a: 0)
}
```
I would have expected no output here.

Reply:
Agree that that's not right. However, in the spirit of our use case here, what I'd expect is actually this:

```
bla::X {
  a: 0
}
```

I'll give that a try.

Reply:
Adding three new commits at the end implementation this. The first two change how (normal) bitfields and also offsets are printed, the 3rd then implements printing of anonymous bitfields.